ext/opcache/jit/zend_jit.c

```diff
@@ -3135,20 +3135,8 @@ static int zend_jit_setup_hot_counters(zend_op_array *op_array)
 
 static int zend_needs_manual_jit(const zend_op_array *op_array)
 {
-	if (op_array->doc_comment) {
-		const char *s = ZSTR_VAL(op_array->doc_comment);
-		const char *p = strstr(s, "@jit");
-
-		if (p) {
-			size_t l = ZSTR_LEN(op_array->doc_comment);
-
-			if ((p == s + 3 || *(p-1) <= ' ') &&
-			    (p + 6 == s + l || *(p+4) <= ' ')) {
-				return 1;
-			}
-		}
-	}
-	return 0;
+	return op_array->attributes &&
+        zend_hash_str_exists(op_array->attributes, "opcache\\jit", sizeof("opcache\\jit")-1);
```

Comment:
Can be simplified to `zend_hash_str_exists(op_array->attributes, ZEND_STRL("opcache\\jit");`